osrf_testing_tools_cpp/src/memory_tools/CMakeLists.txt

```diff
@@ -47,7 +47,7 @@ option(OSRF_TESTING_TOOLS_CPP_DISABLE_MEMORY_TOOLS
 # Memory tools doesn't work on Windows right now.
 # TODO(wjwwood): If on aarch64, so not set the preload environment variables, until fixed.
 # see: https://github.com/osrf/osrf_testing_tools_cpp/issues/3
-if(WIN32 OR CMAKE_SYSTEM_PROCESSOR STREQUAL "aarch64")
+if(WIN32 OR CMAKE_SYSTEM_PROCESSOR STREQUAL "aarch64" OR CMAKE_SYSTEM_PROCESSOR STREQUAL "aarm64")
```

Comment:
Why this change?

Reply:
I think what CMake reports is pretty wildly variable based on a quick search: https://cmake.org/cmake/help/latest/variable/CMAKE_HOST_SYSTEM_PROCESSOR.html#variable:CMAKE_HOST_SYSTEM_PROCESSOR

I think if we have evidence that `uname -m` is returning something that isn't `aarch64`, it would be worth adding to this list.

Reply:
I assumed from that comments that this library should've followed suit in the path of a Unix machine so checking the system processor via the style guide made the most sense. To follow up though, I had accidently added an extra a to arm64 when I made the commit, the CMake file is triggered when `CMAKE_SYSTEM_PROCESSOR STREQUAL "arm64". But, the library will compile reguardless of the processor value, it just relies on a CMake_CXX_Standard update.